packages/react/src/PricingOptions/PricingOptions.module.css

```
@@ -0,0 +1,339 @@
+/* ---------------------------------------------------------- */
+/* Pricing options component                                  */
+/* ---------------------------------------------------------- */
+/* Table of contents:                                         */
+/* 1. Container                                               */
+/* 2. Layout: default                                         */
+/* 3. Layout: cards                                           */
+/* 4. Single item                                             */
+/* 5. Multiple items                                          */
+/* 6. Item inner layout                                       */
+/* 7. Content                                                 */
+/* 8. Feature list                                             */
+/* ---------------------------------------------------------- */
+
+/* ---------------------------------------------------------- */
+/* 1. Container                                               */
+/* ---------------------------------------------------------- */
+
+.PricingOptions {
+  --pricing-cards-item-gap: var(--brand-PricingOptions-items-gap);
+  --pricing-cards-container-padding-inline: var(--brand-PricingOptions-items-container-padding-inline);
+
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto;
+  row-gap: var(--pricing-cards-item-gap);
+  padding-inline: var(--pricing-cards-container-padding-inline);
+}
+
+/* ---------------------------------------------------------- */
+/* 2. Layout: default                                         */
+/* ---------------------------------------------------------- */
+
+.PricingOptions--presentation-default {
+  border-radius: var(--brand-borderRadius-xlarge);
+  border: solid 1px var(--brand-color-border-default);
```

we have a token for `1px` btw.. 

```suggestion
  border: solid var(--brand-borderWidth-thin) var(--brand-color-border-default);
```